### DISCUSSION TITLE
Not working again

### DISCUSSION THREAD

**Discussion by juanclem** [thatskymod/Sky-CotL-Scripts#74] [General]
Sky just _updated_ Dukun isn't working 💔

**Comment by JosieFU** [comment]
essentials, witchcraft, teleport nothing is working😭💔

**Comment by axxstlr** [comment]
Not working Dukunlangit 🥲

**Comment by AlexSkykid070** [comment]
Its still not wprking for me too 😭😭
Hopefully this gets updated/fixed soon

**Comment by Tetr1sForev3r** [comment]
Probably because of the Cinnamoroll update. For now we gotta wait till it gets updated

**Reply by Haki-Skykid** [reply]
Is there any way to contact that sky mod? I want to send him a bug mail so he can quickly fix the error

**Comment by thatskymod** [comment]
## Resolved in Dukun Langit v0.2.3rc

The issue shall now be revolved as of **Dukun Langit v0.2.3rc**.

The file is available for download down below:

https://github.com/thatskymod/Sky-CotL-Scripts/blob/d021c69403acb5cc6f9d705c510bc3f110e47bb7/Canvas/Libs/libLangit023rc.so

**Note for future readers:**
With the rapid pace at which mods are evolving, it is difficult to keep links on aged discussions up-to-date. If you find that the link is broken, consider searching the repository manually.

**Reply by Haki-Skykid** [reply]
❤❤
